### DISCUSSION TITLE
从 v2.6.4 升级到 v2.6.7 之后无法通过编译

### DISCUSSION THREAD

**Discussion by BenzzzX** [xmake-io/xmake#2444] [Q&A] [answered]
xmake 额外生成了 xxx.cpp.cpp，这个文件无法通过编译（目测缺失头文件），并且因为文件插入了很多 #line 反而很难定位到具体错误。

**Comment by waruqi** [comment, marked as answer]
有问题，请开 issues 并提供详细错误，光说编译不过，我还能直接猜出来原因？
